### PR TITLE
Add support for a new RDAP JSON value type

### PR DESCRIPTION
**Pull request by j-bernard**
IANA registry got a new JSON value type which led to an exception. @anewton1998 made a similar fix quite recently.

### CODE REVIEW

File: validator/src/test/java/org/icann/rdapconformance/validator/workflow/rdap/dataset/model/BootstrapDomainNameSpaceTest.java [hunk 1]
```diff
@@ -15,7 +15,6 @@ public class BootstrapDomainNameSpaceTest extends BaseUnmarshallingTest<Bootstra
 
     @BeforeMethod
     public void setUp() throws Throwable {
-        //this.bootstrapDomainNameSpace.parse(getClass().getResourceAsStream("/dataset/dns.json"));;
         this.bootstrapDomainNameSpace = deserializer.deserialize(new File(getClass().getResource("/dataset/dns.json").getFile()));
```
Comment (anewton1998) [inline validator/src/test/java/org/icann/rdapconformance/validator/workflow/rdap/dataset/model/BootstrapDomainNameSpaceTest.java:18]: What is this change for? It seems to be unrelated to a new IANA value.

File: validator/pom.xml [hunk 2]
```diff
@@ -106,7 +105,7 @@
         </dependency>
 
         <dependency>
-            <groupId>com.github.tomakehurst</groupId>
+            <groupId>org.wiremock</groupId>
```
Comment (anewton1998) [inline validator/pom.xml:108]: This seems unrelated. Why is this dependency changing?

File: validator/src/test/java/org/icann/rdapconformance/validator/workflow/rdap/dataset/model/BootstrapDomainNameSpaceTest.java (hunk 1 above)
Comment (j-bernard) [reply validator/src/test/java/org/icann/rdapconformance/validator/workflow/rdap/dataset/model/BootstrapDomainNameSpaceTest.java:18]: Some cleaning I went through

File: validator/pom.xml (hunk 2 above)
Comment (j-bernard) [reply validator/pom.xml:108]: A warning was indicating that the groupId had changed for wiremock so I figured it would be better to use the new one

**Review by anewton1998 — APPROVED** [review]
